HDSAviz/basic_plot.py

```diff
@@ -0,0 +1,122 @@
+"""This is a basic script for plotting an analysis file."""
```

Comment:
Sounds good. Just a heads up I changed the inputs slightly (the minvalues is a float instead of a numpy array) so be sure to check this before running. Also I have default values set so if you just supply a data frame that should work too.